rpcs3/Emu/RSX/RSXFIFO.cpp

```diff
@@ -233,7 +233,11 @@ namespace rsx
 			else
 			{
 				// Return a raw pointer with no limited access
-				return {static_cast<const u32*>(vm::base(m_iotable->get_addr(m_internal_get))), 0x10000};
+				constexpr u32 _1m = 0x100000;
```

Comment:
The caller to get_current_arg_ptr should specify the length they wish to consume.